content/docs/setup/kubernetes/upgrade/index.md

```diff
@@ -25,34 +25,11 @@ and change directory to the new release directory.
 ### Control plane upgrade
 
 The Istio control plane components include: Citadel, Ingress gateway, Egress gateway, Pilot, Galley, Policy, Telemetry and
-Sidecar injector.
+Sidecar injector.  Choose one of the following two **mutually exclusive** options to update the control plane:
 
-#### Helm upgrade
-
-If you installed Istio using [Helm and Tiller](/docs/setup/kubernetes/install/helm/#option-2-install-with-helm-and-tiller-via-helm-install),
-the preferred upgrade option is to let Helm take care of the upgrade:
-
-1. To keep updated all the Istio [Custom Resource Definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions) (CRDs), you must upgrade the `istio-init` chart. If the chart doesn't exist you must install the new release.
-
-    {{< text bash >}}
-    $ helm upgrade --install istio-init install/kubernetes/helm/istio-init --namespace istio-system
-    {{< /text >}}
-
-1. Check that all the CRD creation jobs completed successfully to verify that the Kubernetes API server received all the CRDs:
-
-    {{< text bash >}}
-    $ kubectl get job --namespace istio-system | grep istio-init-crd
-    {{< /text >}}
-
-1. Upgrade the `istio` chart:
-
-    {{< text bash >}}
-    $ helm upgrade istio install/kubernetes/helm/istio --namespace istio-system
-    {{< /text >}}
-
-#### Kubernetes rolling update
-
-You can also use Kubernetes’ rolling update mechanism to upgrade the control plane components.
+{{< tabset cookie-name="controlplaneupdate" >}}
+{{% tab name="Kubernetes rolling update" cookie-value="k8supdate" %}}
```

Comment:
I made an update this morning, for tabs please use {{< instead of {{% from now on.